bpf/cpu/cpu.bpf.c

```diff
@@ -146,28 +152,22 @@ u32 UNWIND_SAMPLES_COUNT = 7;
 /*================================ MAPS =====================================*/
 
 BPF_HASH(stack_counts, stack_count_key_t, u64, MAX_STACK_COUNTS_ENTRIES);
-BPF_STACK_TRACE(stack_traces, MAX_FRAME_POINTER_WALKED_STACKS);
+BPF_STACK_TRACE(stack_traces, MAX_STACK_TRACES);
+BPF_HASH(dwarf_stack_traces, int, stack_trace_t, MAX_STACK_TRACES);
```

Comment:
In the future, we will replace this with a single map for stack traces. I will open an issue to track this effort.

Reply:
Thanks, I was gonna ask if this is temporary. Makes sense!

Reply:
Thanks, I was gonna ask if this is temporary. (Github somehow deleted this comment in between 2 revisions. :laughing: )